lib/topological_inventory/orchestrator/object_manager.rb

```diff
@@ -31,6 +31,8 @@ def delete_pod(name)
 
       def get_deployment_config(name)
         connection.get_deployment_config(name, my_namespace)
+      rescue KubeException
+        nil
```

Comment:
Should we include a logged messge here? Just asking because it stuck out to me.

Reply:
Sure, only reason I didn't is we aren't doing it anywhere else. I'll add it to the rest of the rescues in the future!